Tests/LanguageServerProtocolJSONRPCTests/ConnectionTests.swift

```diff
@@ -54,37 +54,37 @@ class ConnectionTests: XCTestCase {
   func testMessageBuffer() async throws {
     let client = connection.client
     let clientConnection = connection.clientToServerConnection
-    let expectation = self.expectation(description: "note received")
+    let expectation = self.expectation(description: "notfication received")
 
-    await client.appendOneShotNotificationHandler { (note: EchoNotification) in
-      XCTAssertEqual(note.string, "hello!")
+    await client.appendOneShotNotificationHandler { (notification: EchoNotification) in
+      XCTAssertEqual(notification.string, "hello!")
       expectation.fulfill()
     }
 
-    let note1 = try JSONEncoder().encode(JSONRPCMessage.notification(EchoNotification(string: "hello!")))
-    let note2 = try JSONEncoder().encode(JSONRPCMessage.notification(EchoNotification(string: "no way!")))
+    let notification1 = try JSONEncoder().encode(JSONRPCMessage.notification(EchoNotification(string: "hello!")))
+    let notification2 = try JSONEncoder().encode(JSONRPCMessage.notification(EchoNotification(string: "no way!")))
 
-    let note1Str: String = "Content-Length: \(note1.count)\r\n\r\n\(String(data: note1, encoding: .utf8)!)"
-    let note2Str: String = "Content-Length: \(note2.count)\r\n\r\n\(String(data: note2, encoding: .utf8)!)"
+    let notification1Str: String = "Content-Length: \(notification1.count)\r\n\r\n\(String(data: notification1, encoding: .utf8)!)"
```

Comment:
I think these lines exceed the 120 column limit for the repo. Could you run the `Run swift-format` task in VS Code to format the changes? 

---

Also, I think the `: String` annotation isn’t necessary. Since you’re touching this anyway, you can remove it if you want – doesn’t bring it below the 120 character column limit though, unfortunately 😕; same in the line below

Reply:
Yeah sure, I will do that.